### PR TITLE
{bio}[foss/2018b] Porechop v0.2.4 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@sassy-crick Please add the missing `download_dep_fail` and `use_pip`, as reported by the failing tests?

**Comment by sassy-crick** [comment]
@boegel done. Sorry it took so long.

File: easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+source_urls = ['https://github.com/rrwick/Porechop/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['44b499157d933be43f702cec198d1d693dcb9276e3c545669be63c2612493299']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = ['none']
+use_pip = ['none']
```
Comment (boegel) [inline easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-foss-2018b-Python-3.6.6.eb]: @sassy-crick This should be

```python
download_dep_fail = True
use_pip = True
```

I guess I should blame myself for this one, for not having written up proper documentation on all this...

Comment (sassy-crick) [reply easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-foss-2018b-Python-3.6.6.eb]: @boegel no worries. I am not always the best in reading and understanding them... :-)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/ef1583c89b23ebfe8629ab71111f6aff for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @sassy-crick!

**Comment by migueldiascosta** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**